x-pack/plugins/security_solution/public/management/components/console/components/console_manager/types.ts

```diff
@@ -5,24 +5,47 @@
  * 2.0.
  */
 
-import type { ReactNode } from 'react';
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+import type { ComponentType } from 'react';
 import { ConsoleProps } from '../../types';
 
-export interface ConsoleRegistrationInterface<Meta extends object = Record<string, unknown>> {
+export interface ConsoleRegistrationInterface<TMeta extends object = any> {
```

Comment:
why was this change needed?

Reply:
I kept getting TS errors when using this generic with an index interface, which I felt was real weird since any `interface` should be able to meet the requirement of `Record<string, unknown>`. Since the components here don't care about that `meta` is (its just a pass through to caller defined components that are rendered later), I just set it to `any`

Reply:
I thought that might be the reason. So I already did try `Record<string, unknown>` locally before asking you this, and it did not give me any TS errors. Maybe you should give it another try. 🤷

Reply:
I thought that might be the reason. So I already did try `Record<string, unknown>` locally before asking you this, and it did not give me any TS errors. Maybe you should give it another try. 🤷